webpack/assets/javascripts/react_app/components/BreadcrumbBar/components/Breadcrumb.js

```diff
@@ -16,15 +17,26 @@ const Breadcrumb = ({
 
   return (
     <PfBreadcrumb title={title} {...props}>
-      {items.map((item, index) => (
-        <PfBreadcrumb.Item
-          key={index}
-          active={index === items.length - 1}
-          onClick={item.onClick}
-          href={item.url}
-          dangerouslySetInnerHTML={{ __html: item.caption }}
-        />
-      ))}
+      {items.map((item, index) => {
+        const active = index === items.length - 1;
+        const { caption, caption: { icon, text } } = item;
+
+        return (
+          <PfBreadcrumb.Item
+            key={index}
+            active={active}
+            onClick={item.onClick}
+            href={item.url}
+            title={item.caption.text || item.caption}
+          >
+            {icon && <img src={icon} />}
+            {' '}
+            {text || caption}
+            {' '}
+            { active && <span id='title-addition' > {titleAddition} </span>}
```

Comment:
so we have two options I can think of:
1. replace the entire title when fqdn is changed
2. add  this title's addition only on new host page (keeping today's behavior )

Reply:
I prefer 1. It means copying some strings into js code but in general I find it more re-usable.

Reply:
I think it makes sense to do two things - 1. change the entire title instead of adding to it (so if needed we can do that elsewhere easily, where it might not make sense to only add text) 2. only do this on new hosts since it's confusing if you are editing an existing one - you expect the old name to be there rather than the new

Reply:
@tbrisker I agree. Changing it on the edit page would be confusing.